clients/drcachesim/common/options.cpp

```diff
@@ -849,6 +849,17 @@ droption_t<bool> op_enable_kernel_tracing(
     "syscall's PT and metadata to files in -outdir/kernel.raw/ for later offline "
```

Comment:
> This test is run without superuser permissions to show that superuser permissions are not required for other parts of the kernel tracing workflow.

Please clarify that the tracing part ifself does need the paranoid flag set which requires superuser (it is pretty much never going to be set to the value needed by default on any machine due to security risks right?) -- otherwise this makes it sound like I could go get a kernel PT trace on a random machine and the only permission issue I might hit is the kcore.

Reply:
Hmm.. now I'm wondering whether this test should be part of the suite or not. It's currently enabled in our cmake if the system supports PT, but perf_event_paranoid may not be set to a favorable value, and our cmake cannot do that without sudo, and then it's not a non-sudo test anymore.

Maybe I should suffix _SUDO so that it's enabled only when RUN_SUDO_TESTS is set.

Reply:
Done.